napari/plugins/manager.py

```diff
@@ -341,6 +214,67 @@ def _register_module(
         except Exception as exc:
             raise PluginRegistrationError(plugin_name, module_name) from exc
 
+    def register(self, plugin: ModuleType, name=None):
```

Comment:
vendored from pluggy, but contains the changes in my PR that allow the `specname` argument to be used by developers.